multi-layer-perceptron/mxnet_test.py

```diff
@@ -0,0 +1,226 @@
+'''
+Wat.ai Python/Julia framework benchmark comparison Project
+Milestone 1
```

Comment:
Small but I like this top level comment, copied it to my script, just gives a nice overview of what is going on.